ksql-functional-tests/src/test/resources/query-validation-tests/timestampformat.json

```diff
@@ -37,6 +37,38 @@
       "outputs": [
         {"topic": "TS", "value": {"ETS": 1566912669200}, "timestamp": 1566912669200}
       ]
+    },
+    {
+      "name": "KSQL override output timestamp for CSAS",
```

Comment:
would be good to extend this set of tests to include some negative tests, e.g.

 * where `sink_ts` doesn't exist in the schema (add separate test)
 * where `sink_ts` includes invalid data, i.e. where ID is a negative number. (add to existing tests)
* where `sink_ts` is in string form, i.e. use timestampformat in the with of the CTAS/CSAS, (add separate tests)
 * where the source has a `timestamp` column set in the WITH clause, and that column exists in the CSAS schema: the timestamp should NOT be updated in the CSAS, e.g. 

```json
 {
      "name": "timestamp column of source should not influent sink",
     "statements": [
        "CREATE STREAM INPUT (ID bigint, EVENT_TS bigint) WITH (kafka_topic='test_topic', value_format='JSON', timestamp='EVENT_TS');",
        "CREATE STREAM OUTPUT AS SELECT id as EVENT_TS FROM INPUT;"
      ],
      "inputs": [
        {"topic": "test_topic", "value": {"ID": 1, "EVENT_TS":  1526075913000}},
        {"topic": "test_topic", "value": {"ID": 2, "EVENT_TS":  1589234313000}}
      ],
      "outputs": [
        {"topic": "OUTPUT", "value": {"EVENT_TS":1}, "timestamp": 1526075913000},
        {"topic": "OUTPUT", "value": {"EVENT_TS":3}, "timestamp": 1589234313000}
      ]
    }
```

Reply:
Done. Added more tests.